engine/src/main/java/org/camunda/bpm/engine/impl/dmn/transformer/DecisionDefinitionHandler.java

```diff
@@ -25,6 +27,15 @@
 
 public class DecisionDefinitionHandler extends DmnDecisionTransformHandler {
 
+  protected final Integer configuredHistoryTTL;
+  protected Function<String, Integer> parseFunction = ParseUtil::parseHistoryTimeToLive;
+
+  public DecisionDefinitionHandler(String configuredHistoryTTL) {
+    this.configuredHistoryTTL = Optional.ofNullable(configuredHistoryTTL)
+        .map(parseFunction)
+        .orElse(null);
+  }
```

Comment:
🙃 In my opinion this part could be simplified by not using Optionals.
```suggestion
  protected final Integer configuredHistoryTTL;

  public DecisionDefinitionHandler(String configuredHistoryTTL) {
    this.configuredHistoryTTL = ParseUtil.parseHistoryTimeToLive(configuredHistoryTTL);
  }
```

Reply:
My thinking was misguided by ignoring parseHistoryTimeToLive handles null values. I've applied the above suggestion.